sway-lsp/src/capabilities/document_symbol.rs

```diff
@@ -1,10 +1,9 @@
-use crate::core::token::{get_range_from_span, SymbolKind, Token};
+use crate::core::token::{SymbolKind, Token, TokenIdent};
 use lsp_types::{self, Location, SymbolInformation, Url};
-use sway_types::{Ident, Spanned};
 
 pub fn to_symbol_information<I>(tokens: I, url: Url) -> Vec<SymbolInformation>
 where
-    I: Iterator<Item = (Ident, Token)>,
+    I: Iterator<Item = (TokenIdent, Token)>,
```

Comment:
Is the token in TokenIdent the same as the token in the 2nd part of the tuple? A comment here would be helpful.

Reply:
I left a doc comment description that explains the differences between the `sway_types::Ident` the the `TokenIdent`. Since this can be pulled up when hovering over the type, and considering it's used throughout the server, maybe that's ok?